llvm/lib/Transforms/Utils/CodeLayout.cpp

```diff
@@ -62,6 +62,12 @@ cl::opt<bool> ApplyExtTspWithoutProfile(
     "ext-tsp-apply-without-profile",
     cl::desc("Whether to apply ext-tsp placement for instances w/o profile"),
     cl::init(true), cl::Hidden);
+
+namespace codelayout {
+cl::opt<unsigned>
+    CDMaxChainSize("cdsort-max-chain-size", cl::Hidden, cl::init(128),
```

Comment:
Does it make sense to keep and use the new option next to other cdsort options, e.g., `CacheEntries`, `CacheSize`? It would be also great to unify their namings

Reply:
This is a cdsort option, different from ext-tsp. Perhaps place this after all the other `ext-tsp` options?

Reply:
This would be a [place](https://github.com/MaskRay/llvm-project/blob/9403e71c4270bf830384b43b5c8deff13e1bf3d3/llvm/include/llvm/Transforms/Utils/CodeLayout.h#L63) to add the option

Reply:
Thanks! I figured it out. 

Other options are still named `cds-*`, but this new one is named `cdsort-max-chain-size`. I think it makes sense to reduce the number of abbreviations. As we have used `cdsort` and various case variants, it makes sense to avoid `cds-*`.

Perhaps land a separate commit to rename existing `cds-*` options to `cdsort-*`?

Reply:
Yes, that was exactly my motivation for the earlier comment on "unifying their namings"; sorry for such brevity